### DISCUSSION TITLE
Waves v12 plugins don't work

### DISCUSSION THREAD

**Discussion by jaacMusic** [apohl79/audiogridder#259] [Q&A] [answered]
The new version of Wave v12 plugins doesn't load. AG server detects them but it won't show on the AG plugin side. After a few AG server restart, they appear as failed.

**Comment by CaBuffalo** [comment]
I haven't tried V12 on AG. But it sounds like the problem I have on IK Multimedia plugins.

**Comment by apohl79** [comment]
They do work without issues on macOS 10.14 and 10.15 on my personal setup. @jaacMusic: You are running a Win10 Server?

**Comment by jaacMusic** [comment]
They show in AG server but from the AG plugin, they don't show. I'm running Windows 10 for both DAW and Server.

**Comment by apohl79** [comment]
I will check if I can see this too.

**Comment by CaBuffalo** [comment]
I'll run a test later today as well.

**Comment by jaacMusic** [comment]
Sorry, I haven't had a chance to retest but I'll do it after work hours. Thanks!

**Comment by jaacMusic** [comment]
I did some testing with an empty project and I still don't get to see Waves v12 plugins (checked both VST and VST3 for them).
Also note, that in the AG server available plugins window it shows version 1.0.0.0 for all Waves plugins while others that do work show the proper version. Could it be incompatibility with how AG server reads/load the plugins? See screenshot below:

![image](https://user-images.githubusercontent.com/68957086/102692557-1d01be80-41e2-11eb-9191-5c4f1a397bd6.png)

**Comment by apohl79** [comment]
Can you attach the server and client logs. For some reason, the server does not include the plugins for the client channel config.

**Comment by jaacMusic** [comment]
I did a bit more testing a VST plugins are loading under Effect category. They both show in AG server and AG plugin.
VST3s are not detected at all by the AG server for some reason. I'll provide the client and server shortly.

**Comment by jaacMusic** [comment, marked as answer]
I think I figured it out and it is working now. It just happens that Waves v12 has included in the instances of the VST3 file their instruments which I did not install. So the solution was to rescan and wait for every time it asked for the location of the instrument and cancel it and click wait on AG server every time it showed until it completed going over all Waves instruments. It took like 5 minutes clicking wait and canceling until it finished and then...boommm they appeared! Thanks!!!

**Reply by CaBuffalo** [reply]
NICE !!!
